Mobile Buy SDK/Mobile Buy SDK/Models/Persistent/BUYProduct.m

```diff
@@ -118,12 +118,15 @@ - (BOOL)isDefaultVariant
 {
 	if ([self.variants count] == 1) {
 		BUYProductVariant *productVariant = [self.variants firstObject];
-		BUYOptionValue *optionValue = [productVariant.options anyObject];
-		NSString *defaultTitleString = @"Default Title";
-		NSString *defaultString = @"Default";
-		if ([productVariant.title isEqualToString:defaultTitleString] &&
-			([optionValue.value isEqualToString:defaultTitleString] || [optionValue.value isEqualToString:defaultString])) {
-			return YES;
+		if ([productVariant.options count] == 1) {
+			BUYOptionValue *optionValue = [productVariant.options anyObject];
+			NSString *defaultTitleString = @"Default Title";
```

Comment:
Let's make these proper constants. Define them outside of the method scope. If they are private, make sure to declare them static:

`static NSString * const <name> = <value>;`